wifiphisher/common/phishinghttp.py

```diff
@@ -46,26 +46,24 @@ def get(self, url):
                 log_file.write('[' + T + '*' + W + '] ' + O + "GET" + W + " request from " + T +
                                self.request.remote_ip + W + " for " + self.request.full_url() +
                                "\n")
-                log_file.close()
         # Ignore weird requests
         except:
             pass
 
-    def post(self, url):
+    def post(self):
         form_data = self.request.body.split('&')
         form_values = []
         for each_form_data in form_data:
             form_values.append(each_form_data.split('='))
         for input_name,input_value in form_values:
```

Comment:
What identical loops do you mean?

The first iteration has to finish before the second one is iterated, or are you talking about something else?